.github/workflows/ios.yml

```
@@ -118,58 +114,72 @@ jobs:
+          }
+          const dts = data.devicetypes || [];
+          let dtId = (dts.find(dt => dt.name==='iPhone 16 Pro') || dts.find(dt => (dt.name||'').startsWith('iPhone')) || {}).identifier || '';
+          let devId = '';
+          if (rtId && dtId) {
+            const devs = (data.devices[rtId]||[]).filter(dev => dev.isAvailable && dev.deviceTypeIdentifier===dtId);
+            if (devs.length) devId = devs[0].udid;
+          }
+          console.log(rtId);
+          console.log(dtId);
+          console.log(devId);
+NODE
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 156: could not find expected ':'

(syntax-check)